src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -86,7 +86,7 @@ public HubConnectionContext(ConnectionContext connectionContext, TimeSpan keepAl
 
                 Interlocked.Exchange(ref _lastSendTimestamp, Stopwatch.GetTimestamp());
 
-                await _connectionContext.Transport.Output.FlushAsync();
+                await _connectionContext.Transport.Output.FlushAsync(ConnectionAbortedToken);
```

Comment:
On `HubConnectionContext` there is a virtual `ConnectionAbortedToken` token, but also a private `_connectionAbortedTokenSource` that is used in multiple places. What is `ConnectionAbortedToken` for and why is it virtual?

Reply:
Thoughts on this @anurse @davidfowl